apps/sample-imdb/components/movies-list.tsx

```diff
@@ -1,59 +1,79 @@
-'use client'
-
-import { FC, use, useMemo } from 'react'
+import Link from 'next/link'
 import { Titles } from '~/lib/xata.codegen.server'
+import { Rating } from './ratings'
+
+const url = process.env.VERCEL_URL
 
-const search = async (term?: string) => {
-  const response = await fetch(`/api/search?term=${term}`)
+const search = async (term?: string): Promise<Titles[]> => {
+  const response = await fetch(`${url}/api/search?term=${term}`)
+  const { titles } = await response.json()
 
-  return response.json()
+  return titles
```

Comment:
Careful, this is `any`, should we consider using `trpc` to improve this? (not on this PR obviously ^^)

Reply:
this is gone. Not making client-side requests anymore...